mypy/test/testsamples.py

```diff
@@ -0,0 +1,49 @@
+"""Self check mypy package"""
+import sys
+import os.path
+from typing import List, Set
+
+from mypy.test.helpers import Suite, run_mypy
+
+
+class SamplesSuite(Suite):
+    def test_stubs(self) -> None:
+        # We only test each module in the one version mypy prefers to find.
+        # TODO: test stubs for other versions, especially Python 2 stubs.
+        seen = set()  # type: Set[str]
+        modules = []
+        # TODO: This should also test Python 2, and pass pyversion accordingly.
+        for version in ["2and3", "3", "3.5"]:
+            # FIX: remove 'builtins', this directory does not exist
```

Comment:
What does "FIX:" mean? Is it a TODO? Why not do it now?

Reply:
I guess you are right. I think I didn't want to change anything